### PR TITLE
Remove Governikus.AusweisApp2 1.22.6

### PR DESCRIPTION
**Pull request by Trenly**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/74120)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/g/Governikus/AusweisApp2/1.22.6/Governikus.AusweisApp2.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://www.ausweisapp.bund.de/fileadmin/user_upload/Software/AusweisApp2-1.22.6.msi
```
Comment (ItzLevvie) [inline manifests/g/Governikus/AusweisApp2/1.22.6/Governikus.AusweisApp2.installer.yaml:15]:
```
  InstallerUrl: https://www.ausweisapp.bund.de/fileadmin/user_upload/Software/AusweisApp2-1.22.6.msi
```
404 Not Found

Comment (ItzLevvie) [reply manifests/g/Governikus/AusweisApp2/1.22.6/Governikus.AusweisApp2.installer.yaml:15]: image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
Waiting for the new version to be merged before converting this to a removal

**Comment by Trenly** [comment]
@ItzLevvie - Converted to a removal PR now that newer version has been merged in

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**